apps/fluent-tester/src/TestComponents/Test.tsx

```diff
@@ -155,6 +155,7 @@ export const Test = (props: TestProps): React.ReactElement<Record<string, never>
         )}
       </Stack>
       {props.sections.map((section, index) => {
+        if (section == null) return <></>;
```

Comment:
style nit: For readability, it'd be nice to:
1. Have a newline after this conditional statement
2. Have the return statement be on its own line.

Having it on one line like this without any breaks makes this easy to miss while reading code.